### PR TITLE
Php8 fixes

### PR DESCRIPTION
**Pull request by digitalchild**
### All Submissions:

More php8 fixes. 

### How to test the changes in this Pull Request:

1. This removes warnings and fatal errors related to dates in the existing reports. 
2. Also re-enables the import/export tool for admins.

### CODE REVIEW

**Comment by mahlamusa** [comment]
Maybe we should separate the Import & Export because it does not seem to work properly.

When importing as a vendor:
- Importing and selecting to update the existing products. It reports that the products that products were updated but no products are shown for the current vendor, but visible when viewing as admin
- Importing and not choosing to update existing products: It reports products were successfully imported but no products are shown for the vendor but are added to the admin.

Basically it seems the product owner check is somehow broken, it should report an error if the vendor does not own the product or import the products and assign them to the vendor. Debug logs added seem to indicate no issues in the code, maybe its something else

**Review by mahlamusa — APPROVED** [review]
The PHP8 fix works. Also the import seems to work with the shared CSV.
